server/src/main/resources/databasequery/additionalInformation.html

```diff
@@ -70,7 +70,7 @@ <h2>Parameter Substitution</h2>
 
   <p>
     <code>
-      select * from Results r where eventId = :event_id and personId =
+      select * from Results r where event_id = :event_id and person_id =
```

Comment:
```suggestion
      select * from results r where event_id = :event_id and person_id =
```

Reply:
I think MySQL is a bit forgiving when it comes to the casing of table names, but for consistency's sake we should accurately reflect the changes.